### PR TITLE
Added basic readme

### PR DESCRIPTION
**Pull request by philipheinisch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,28 @@
 
 [![License](https://licensebuttons.net/l/by-nc-nd/3.0/88x31.png)](https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode)
 
+ToolkitICL is an open source tool useful for automated execution
```
Comment (ranocha) [inline README.md]: I would delete that sentence and add it as description at the top of this project (with a "." at the end).

File: README.md [hunk 2]
```diff
@@ -2,7 +2,28 @@
...
 
+ToolkitICL is an open source tool for automated OpenCL kernel execution. It can be used as an easy to use cross platform tool to execute a set of kernels for example on compute clusters, run automated OpenCL benchmarks or test and validate kernels. HDF5 files are used for the entire configuration and data handling. The list of kernels to be executed, all variables and the workgroup size is defined in the input HDF5 file. After execution, the output data, data copy- and runtime is written to the output HDF5 file.
+
+## Usage
+To use toolkitCL the following needs to be installed:
+- OpenCL (headers and drivers)
+- HDF5
+- CMake
+
+In case some of the OpenCL headers ae missing, they can be obtained directly from Khronos: [https://github.com/KhronosGroup](https://github.com/KhronosGroup).
```
Comment (ranocha) [inline README.md]: "are" missing

File: README.md [hunk 3]
```diff
@@ -2,7 +2,28 @@
...
+On some compute cluster systems, it might be necessary to explicitly define the library path in the cmake configuration files. If no icd loader is available, the path to the OpenCL vendor library hast to be defined manually. 
+To compile the code using Linux, create a build directory and run `cmake ..` followed by `make`. Recent versions of Visual Studio should automatically detect the cmake configuration files and build toolkitICL automatically. Otherwise `cmake-gui` can be used on Windows systems to create Visul Studio project files.
+
+Examples on how to create the input HDF5 files are provided in `notebooks` .
```
Comment (ranocha) [inline README.md]: ```the directory [`notebooks`](https://github.com/IANW-Projects/toolkitICL/tree/master/notebooks)```?

File: README.md [hunk 4]
```diff
@@ -2,7 +2,28 @@
...
+
+ToolkitICL can be controlled by the following command line options:
+- -d device_id: Use the device specified by `device_id`.
```
Comment (ranocha) [inline README.md]: Maybe we should use backticks for the options, e.g. ``` `-d device_id` ```.

**Review by ranocha — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (philipheinisch) [reply README.md]: Indeed, this sentence is redundant and seems to be a remnant of the past...

File: README.md (hunk 3 above)
Comment (philipheinisch) [reply README.md]: In this case should we also insert a link to cmake for Windows?

Comment (ranocha) [reply README.md]: No, that's probably not necessary. But you can do that if you wish.

**Comment by ranocha** [comment]
There are some conflicts because of #1.

**Review by ranocha — APPROVED** [review]

**Pull request merged**